aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/AggregatingProfiler.java

```diff
@@ -17,14 +18,23 @@
 
     private final CopyOnWriteArrayList<AggregatingReporter> profilerReporters = new CopyOnWriteArrayList<>();
 
-    private final Map<String, IndicationProvider> indicators = new ConcurrentHashMap<>();
+    private final Map<String, IndicationProviderTagged> indicators = new ConcurrentHashMap<>();
+    private volatile Tagger tagger;
+
+    public AggregatingProfiler(Tagger tagger) {
+        this.tagger = tagger;
+    }
+
+    public AggregatingProfiler() {
+        this(new NullTagger());
```

Comment:
It would be more convenient  to name no operation tagger as NoopTagger()  in a same way as the NoopProfiler and NoopProfiledCall.